onnxruntime/core/providers/openvino/backend_manager.h

```diff
@@ -39,6 +39,8 @@ class BackendManager {
 
   bool ModelHasSymbolicInputDims(const onnxruntime::GraphViewer& subgraph) const;
   bool ModelHasBatchedInputs(const ONNX_NAMESPACE::ModelProto& model_proto) const;
+  void ValidateInputShapes(const shape_t& shape,
```

Comment:
ValidateInputShapes do not have a string argument in declaration but in definition there is one. Is it by design.